frontend/src/views/VotePage.vue

```diff
@@ -1,52 +1,29 @@
 <template>
-  <form v-if="election !== null" @submit.prevent="voteInElection()">
-    <h1>{{ election.title }}</h1>
-    <p>{{ election.description }}</p>
-    <div v-for="(question, i) in election.questions" :key="i">
-      <p>Question #{{ i + 1 }}</p>
-      <h3>{{ question.question }}</h3>
-      <div v-for="(answer, ai) in question.answers" :key="ai">
-        <input
-          required
-          type="radio"
-          :id="`${i}.${ai}`"
-          :name="`answer.${i}`"
-          :value="`${question.id}.${answer.id}`"
-          v-model="votes[i].selected"
-        />
-        <label :for="`${i}.${ai}`">{{ answer.text }}</label>
-        <br />
-      </div>
-    </div>
-    <button type="submit" class="submit">Send votes</button>
-  </form>
+  <election-vote-form v-if="election" :election="election" @votesSubmitted="voteInElection"></election-vote-form>
```

Comment:
I wonder if self-closing component would also work here

Reply:
I'll try all these suggestions in the next PR. :+1: